cmd/policy-assistant/pkg/cli/analyze.go

```diff
@@ -118,10 +124,18 @@ func RunAnalyzeCommand(args *AnalyzeArgs) {
 	// 3. read example policies
 	if args.UseExamplePolicies {
 		kubePolicies = append(kubePolicies, netpol.AllExamples...)
+
+		// remove and add appropriate examples like v1 policies
+		kubeANPs, _ = loadAndParse[v1alpha1.AdminNetworkPolicy]("./../../../../conformance/base/admin_network_policy/core-gress-rules-combined.yaml")
```

Comment:
Instead of parsing yaml here, it'd probably be better (in the sense of: simpler, easier to maintain) to create these objects in golang.